ibm_watson/speech_to_text_v1.py

```diff
@@ -535,7 +604,10 @@ def create_job(self,
                    customization_id=None,
                    grammar_name=None,
                    redaction=None,
+                   processing_metrics=None,
+                   processing_metrics_interval=None,
                    content_type=None,
+                   audio_metrics=None,
                    **kwargs):
```

Comment:
Is this a breaking change? Should `content_type` be before `processing_metrics`?

Reply:
Good catch!

Reply:
`audio_metrics` is the new param which should be placed after `content_type`.

Reply:
aren't `processing_metrics` and `processing_metrics_interval` new as well?

Reply:
Oh good catch!